site/src/Consumer/WebNotificationConsumer.php

```diff
@@ -15,27 +18,78 @@
  */
 class WebNotificationConsumer extends AbstractNotificationConsumer
 {
+
+    /**
+     * @var NotificationManager
+     */
+    protected $notificationManager;
+
+    /**
+     * @var UserManagerInterface;
+     */
+    protected $userManager;
+
     /**
      * WebNotificationConsumer constructor
      *
-     * @param LoggerInterface $logger
+     * @param LoggerInterface     $logger
+     * @param NotificationManager $notificationManager
      */
-    public function __construct(LoggerInterface $logger)
-    {
+    public function __construct(
+        LoggerInterface $logger,
+        NotificationManager $notificationManager,
+        UserManagerInterface $userManager
+    ) {
         parent::__construct($logger);
+        $this->notificationManager = $notificationManager;
+        $this->userManager = $userManager;
+    }
+
+    /**
+     * Should a notification be handled by this Consumer?
+     *
+     * @return bool
+     */
+    protected function shouldHandle(): bool
+    {
+        $type = $this->notification->getType();
+        switch ($type) {
+            case "new_project":
+                return true;
+            default:
+                return false;
+        }
     }
 
     /**
      * Handles Notifications that will be sent to UI
      *
      * @return bool
+     *
+     * @throws OptimisticLockException
      */
     protected function handle(): bool
     {
         // TODO: Process the Notification($this->notification)
         echo "Web Notification Consumer";
 
-        return true;
+        $notification = new Notification();
+
+        // Populate the Notification Entity
+        $notification->setMessage($this->notification->getMessage());
+        $notification->setType($this->notification->getType());
+        $notification->setSubject($this->notification->getSubject());
+        $notification->setDate($this->notification->getCreatedAt());
+        $notification->setRecipient($this->notification->getRecipient());
+
+        if ($this->shouldHandle()) {
+            // Persist a Notification
+            $this->notificationManager->addNotification(array($this->notification->getRecipient()), $notification, true);
```

Comment:
Please keep this line around 100 characters.

The newer way of writing array() is just using []